wire-ios-data-model/Tests/Source/Model/Messages/ZMClientMessageTests+Editing.swift

```diff
@@ -608,6 +608,7 @@ extension ZMClientMessageTests_Editing {
         self.performPretendingUiMocIsSyncMoc {
             newMessage = ZMClientMessage.createOrUpdate(from: updateEvent!, in: self.uiMOC, prefetchResult: nil)
         }
+        // TODO: why is `newMessage` never read?
```

Comment:
@netbe Could you please check if the test is correct?

Reply:
updated code to assert it's equal to lastMessage of conv